elasticjob-infra/elasticjob-registry-center/src/test/java/org/apache/shardingsphere/elasticjob/reg/zookeeper/ZookeeperElectionServiceTest.java

```diff
@@ -29,9 +29,12 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
+import org.mockito.internal.verification.VerificationModeFactory;
```

Comment:
Should we use the Class define under package `internal`?
Maybe using `org.mockito.Mockito#atLeastOnce` is better.

Reply:
you are right, I will correct it.